src/xo-app/dashboard/overview/index.js

```diff
@@ -104,10 +104,17 @@ class MissingPatchesPanel extends Component {
     this.setState({ missingPatches })
   }
 
-  _installAllMissingPatches = () => (
-    Promise.all(
-      map(this._getHosts(), installAllHostPatches)
-    )
+  _installAllHostPatches = host => (
+    installAllHostPatches(host).then(() => (
+      getHostMissingPatches(host).then(patches => {
```

Comment:
You could create a `_refreshHostMissingPatches()` method to mutualize some code.